orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+- name: orc8r | Ensure orc8r helm repo
+  command: helm repo add orc8r {{ orc8r_helm_repo }}
+
+- name: orc8r | Update repo metadata
+  command: helm repo update
+
+- name: orc8r | Ensure namespace
+  command: kubectl create ns {{ magma_namespace }}
+  failed_when: false
+
+- name: orc8r | Ensure orc8r dirs
+  file:
+    path: "{{ item }}"
+    state: directory
+    recurse: yes
+  with_items:
+    - "{{ orc8r_config_dir }}"
+    - "{{ orc8r_charts_dir }}"
+    - "{{ orc8r_certs_dir }}"
+
+- name: orc8r | create certificates
+  include_tasks: certs.yml
+
+- name: orc8r | Template out helm chart values
+  template:
+    src: orc8r-values.yaml.j2
+    dest: "{{ charts_dir }}/{{ item }}-values.yaml"
+  with_items:
+    - orc8r
+
+- name: orc8r | template DB setup script
+  template:
+    src: db_setup.sql.j2
+    dest: "{{ orc8r_config_dir }}/db_setup.sql"
+    mode: 600
+
+- name: orc8r | copy orc8r settings secrets
+  copy:
+    src: "{{ item }}"
+    dest: "{{ orc8r_config_dir }}/{{ item }}"
+    mode: 0600
+  with_items:
+    - orc8r-configs.yaml
+    - orc8r-envdir.yaml
+
+- name: orc8r | create orc8r settings secrets
+  command: kubectl -n {{ magma_namespace }} apply -f {{ orc8r_config_dir }}/{{ item }}
+  with_items:
+    - orc8r-configs.yaml
+    - orc8r-envdir.yaml
+  
+- name: orc8r | run DB setup script
+  shell: kubectl -n mariadb exec -i mariadb-0 -- mysql -u root "--password={{ db_root_password }}" mysql < db_setup.sql
+  args:
+    chdir: "{{ orc8r_config_dir }}"
+
+- name: orc8r | precreate persistentvolumeclaims for components
+  script: make_magma_pvcs.sh {{ magma_namespace }} {{ storage_class }} ReadWriteMany {{ item.key }} {{ item.value }}
+  with_items: "{{ magma_pvcs | dict2items }}"
+
+- name: orc8r | Install/upgrade helm charts
+  command: >-
+    helm upgrade --install 
+    -n {{ magma_namespace }}
+    {{ item }}
+    orc8r/{{ item }}
+    {% if orc8r_chart_version is defined %} --version {{ orc8r_chart_version }}{% endif %}
+    -f {{ charts_dir }}/{{ item }}-values.yaml 
+    --wait
+  with_items:
+    - orc8r
+
+- name: orc8r | Get orc8r pod
```

Comment:
The controller pod no longer exists in v1.4. We should set `-l app.kubernetes.io/component=controller` to -l app.kubernetes.io/component=orchestrator`.

Reply:
@mpgermano 1.4 is not officially released still. This logic deploys orc8r/nms version 1.3